show/interfaces/__init__.py

```diff
@@ -319,6 +319,52 @@ def expected(db, interfacename):
 
     click.echo(tabulate(body, header))
 
+# 'mpls' subcommand ("show interfaces mpls")
+@interfaces.command()
+@click.argument('interfacename', required=False)
+@multi_asic_util.multi_asic_click_options
+def mpls(interfacename, namespace, display):
+    """Show Interface MPLS status"""
+
+    ctx = click.get_current_context()
+
+    appl_db = SonicV2Connector()
```

Comment:
use the "pass_db" decorator to access 'db'. Look at other CLIs as an example.

Reply:
@smahesh This will not work since output comes from APPL_DB, not CONF_DB.

Reply:
can you resolve the 'show' as discussed in the e-mail.